### DISCUSSION TITLE
API GetObject Error Response (Golang aws SDK)

### DISCUSSION THREAD

**Discussion by Lunelun** [minio/minio#15672] [Q&A] [answered]
I use api GetObject for loading file size 4.6MB and api return 502 but i use this api download file size 20kb api return file normally.
What it cause ? Is it cause from too large file size and if it cause from too large file size how i can config minio server for solve this problem?

**Comment by krisis** [comment, marked as answer]
`502` response status code suggests that you have a load balancer or reverse proxy in front of your MinIO deployment. Please check your load balancer configuration for any setting that may be preventing downloads of objects larger than a certain size.
